tests/dialects/test_postgres.py

```diff
@@ -1289,3 +1289,6 @@ def test_array_length(self):
                 "clickhouse": UnsupportedError,
             },
         )
+
+    def test_timestamp_with_interval(self):
+        pass
```

Comment:
This belongs to the other PR! 😄

Reply:
Removed it. I am working on one of the open POSTGRES issues related to parathesis and INTERVAL.